docs/var.tfvars-doc.md

```diff
@@ -364,7 +364,7 @@ This variable is used to enable SNAT for OCP nodes. When using SNAT, the OCP nod
 setup_snat                 = true
 ```
 
-These set of variables are specific for CSI Driver configuration and installation.
+**IMPORTANT**: This is an **experimental** feature and not yet ready for production. These set of variables are specific for CSI Driver configuration and installation.
```

Comment:
Please mention the note after the description for better context.

Reply:
Sure